.pipelines/pipeline.yaml

```diff
@@ -261,6 +261,17 @@ stages:
         k8sVersion: ""
         dependsOn: "containerize"
 
+    # Cilium Nodesubnet E2E tests
```

Comment:
I recommend breaking this to two separate PRs. One about CNS changes and one regarding the pipeline for two reasons:
1- This will make the roll back easier in case we encounter some issues.
2- It makes the reviews easier as well.

Reply:
Moved this and other pipeline-related changes out.